packages/core/src/state-management/state-manager.ts

```diff
@@ -432,19 +434,19 @@ export class StateManager {
               await this._updateStateIfPinned(state$)
               return
             }
-            case AnchorStatus.PROCESSING: {
+            case RequestStatusName.PROCESSING: {
               if (!asr.cid.equals(state$.tip)) return
               state$.next({ ...state$.value, anchorStatus: AnchorStatus.PROCESSING })
               await this._updateStateIfPinned(state$)
               return
             }
-            case AnchorStatus.ANCHORED: {
-              await this._handleAnchorCommit(state$, asr.cid, asr.anchorCommit, asr.witnessCar)
+            case RequestStatusName.COMPLETED: {
+              await this._handleAnchorCommit(state$, asr.cid, asr.anchorCommit.cid, asr.witnessCar)
```

Comment:
what happened here?  Did the response format from the CAS change?  Or have we really never been properly processing the anchor commit CID from the anchor service response ever since the beginning of time?

Reply:
It's not the CAS response has changed. We translated CAS Response into an internal representation. And we used just `anchorCommit.cid` part.

Reply:
https://github.com/ceramicnetwork/js-ceramic/pull/2838/files#diff-7920408ff457031025a51c6c64241f297dc91febb1a810e724475548c154c781L245

Reply:
oh I see.  Weird.  We had the full anchor commit body but were throwing out most of it early in the pipeline.  Now we actually pass the full object through.  thanks for clarifying.